1.7/administration/installing/cloud/gce.md

```diff
@@ -22,10 +22,10 @@ A bootstrap node is required to run the scripts and to bootstrap the DC/OS clust
 
 1.  Create the bootstrap node using the Google Cloud Console. In these examples we used a [n1-standard-1](https://cloud.google.com/compute/docs/machine-types) instance running CentOS 7 with a 10 GB persistent disk in `zone europe-west1-c`. The bootstrap node must have "Allow full access to all Cloud APIs" in the Identity and API access section. Also enable Block project-wide SSH keys in the SSH Keys section. Create the instance.
 
-1.  After creating the micro instance, start the instance and run the following from the shell. These commands install prerequisite software on your bootstrap node.
+1.  After creating the boot instance, start the instance and run the following from the shell. These commands install prerequisite software on your bootstrap node.
 
     ```bash
-    $ gcloud components update &&
+    $ sudo gcloud components update &&
```

Comment:
I believe that best practices are to have gcloud installed with user privs, so running the components update as root should not be necessary nor desirable, as that would mutate the privileges of the user installed gcloud instance.